0009-enpoint-events-ipc/README.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Endpoint events IPC
+shortname: 9/ENDPOINT_EVENTS
+status: draft
+editor: Volodymyr Tkhir <vtkhir@cybervisiontech.com>
+---
+
+* [Introduction](#introduction)
+* [Connectivity events](#connectivity-events)
+  * [Endpoint connected](#endpoint-connected)
+  * [Endpoint disconnected](#endpoint-disconnected)
+* [Lifecycle events](#lifecycle-events)
+  * [Endpoint registered](#endpoint-registered)
+  * [Endpoint updated application version](#endpoint-updated-application-version)
+* [Metadata events](#metadata-events)
+  * [Endpoint metadata updated](#endpoint-metadata-updated)
+* [Configuration events](#configuration-events)
+  * [New endpoint configuration available](#new-endpoint-configuration-available)
+  * [Endpoint applied the configuration](#endpoint-applied-the-configuration)
+
+
+## Introduction
+
+The Endpoint events IPC protocol is a [Broadcast Messaging](/0003-messaging-ipc/README.md#broadcast-messaging) protocol extension.
```

Comment:
I wouldn't call these a protocol extension. Rather, I would say that endpoint events are used to communicate events that occur with endpoints (endpoint is the target entity type in terms of 3/MIPC).